### DISCUSSION TITLE
[4.1] Class in the body tag when the tmpl=component is used in the link

### DISCUSSION THREAD

**Discussion by impressionestudio** [joomla/joomla-cms#37419] [2. Cold Features]
Is it possible to have a class in the body tag of the page when the part "tmpl=component" is used in a link? 
This way we will be able to apply a different style to the articles that are displayed alone without the header, footer etc.

**Comment by brianteeman** [comment]
Just add it to the component.php of your template

**Comment by impressionestudio** [comment]
And if I use the Cassiopeia template?

**Reply by drmenzelit** [reply]
You will need to create a child template and override component.php
